pages/docs/react/latest/beyond-jsx.mdx

```diff
@@ -134,18 +120,18 @@ Whenever we are using JSX with a custom component ("capitalized JSX"), we are ac
 <CodeTab labels={["JSX", "Without JSX"]}>
 
 ```res
-<Friend name="Fred" age=1 />
+<Friend name="Fred" age=20 />
 ```
 ```res
-React.createElement(Friend.make, Friend.makeProps(~name="Fred", ~age=1, ()))
+React.createElement(Friend.make, {name: "Fred", age:20})
```

Comment:
Should the output in automatic mode be described here, too?

Reply:
Fair point to me.

Reply:
Added outputs in the automatic mode https://github.com/rescript-association/rescript-lang.org/pull/593/commits/919ef25768b7b35f441e7c7a2c12609fefd8637e